### PR TITLE
Add yarn audit to workflow

### PR DESCRIPTION
**Pull request by geoffharcourt**

### CODE REVIEW

**Review by davidrunger-cl — APPROVED** [review]
Without tests, of course it is difficult to be confident that bumping these JavaScript packages doesn't break anything, but I guess it's probably okay to merge this and see if anything breaks when we start using this.

One thing I am wondering about is the `dist/index.js` file. Does that get used? I cannot see that it does?, since it doesn't seem to be referenced in the `package.json`, except as a build output destination. If it's not used, then I'm wondering if we should delete that file? If it _is_ used, then, in order for the JavaScript package updates to have any effect, we probably would need to rebuild that `dist/index.js` file (which I guess could be done with this repo's `yarn package` command) and commit that diff?

My other question is whether, after merging this, we are going to create a new version tag for this action and start using that in the `commonlit` repo?

**Review by bkjohnson — APPROVED** [review]

**Review by eaierstuck — APPROVED** [review]

**Comment by geoffharcourt** [comment]
@davidrunger-cl messaged you about this in Slack

**Pull request merged**
